public/preload.js

```diff
@@ -16,6 +16,26 @@ Sentry.init({
 })
 
 process.once('loaded', () => {
+  const oldAccessDataRaw = localStorage.getItem('wallet:accessData');
+  if (oldAccessDataRaw) {
+    // When migrating old wallets initialized with hardware wallets we need to clean the localStorage
```

Comment:
Can you add here the version that you are migrating this data from?

Reply:
Done